http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/GKEInterpreter.scala

```diff
@@ -787,8 +813,14 @@ class GKEInterpreter[F[_]: Parallel: ContextShift: Timer](
       raw"""galaxy.rbac.enabled=false""",
       raw"""galaxy.rbac.serviceAccount=${ksa.value}""",
       raw"""rbac.serviceAccount=${ksa.value}""",
-      // TODO Update during https://broadworkbench.atlassian.net/browse/IA-2171
-      raw"""persistence={}"""
+      raw"""persistence.nfs.name=${config.galaxyAppConfig.nfsPersistenceName}""",
+      raw"""persistence.nfs.persistentVolume.extraSpec.gcePersistentDisk.pdName=${nfsDisk.name.value}""",
+      raw"""persistence.nfs.size=${nfsDisk.size.gb.toString}""",
+      raw"""persistence.postgres.name=${config.galaxyAppConfig.postgresPersistenceName}""",
+      raw"""persistence.postgres.persistentVolume.extraSpec.gcePersistentDisk.pdName=${appName.value}-${config.galaxyAppConfig.postgresDiskNameSuffix}""",
```

Comment:
Wonder if we could re-use the above method somehow so as to not duplicate this logic again